Article/Article.js

```diff
@@ -1,20 +1,24 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
+  constructor(article) {
+    
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.article = article
+    
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = article.querySelector('.expandButton');
```

Comment:
```suggestion
    this.expandButton = this.article.querySelector('.expandButton');
```
Since you created a reference to the article passed in and set it to this.article I would use the reference of this.article instead of article which was passed in, both ways work though